cmd/controller-manager/app/options/validation.go

```diff
@@ -1,84 +1,38 @@
 package options
 
 import (
-	"fmt"
-
 	"github.com/karmada-io/karmada/pkg/util"
+
+	"k8s.io/apimachinery/pkg/util/validation/field"
```

Comment:
Could you please group the imports according to `std packages` --> `third-party packages` --> `karmada packages`?

```suggestion
	"k8s.io/apimachinery/pkg/util/validation/field"
	
	"github.com/karmada-io/karmada/pkg/util"
```